automation_tools/__init__.py

```diff
@@ -2201,7 +2201,7 @@ def product_install(distribution, certificate_url=None, selinux_mode=None, sat_v
     installer_options.update({'katello-enable-ostree': 'true'})
 
     # enable async ssh for rex
-    if sat_version not in ('6.3', '6.4', '6.5', '6.6'):
+    if sat_version not in ('6.3', '6.4', '6.5', '6.6', '6.7'):
```

Comment:
wrong

Reply:
it is meant to be 6.7+

Reply:
you mean I have to add 6.8 as well?

Reply:
no , read this as if sat_version is above 6.6

Reply:
enable async ssh rex for 6.7 and 6.8 
negative condition is better as you dont have to add 6.7 then 6.8 then 6.x 
`if sat_version not in [6.3, 6.4, 6.5, 6.6]`
vs.
`if sat_version in [6.7, 6.8]`

Reply:
@lpramuk I read your comment after I pushed new changes. :|